package.json

```diff
@@ -33,19 +33,19 @@
     "prepack": "npm run build"
   },
   "devDependencies": {
-    "@balena/lint": "^5.0.4",
-    "@types/chai": "^4.2.11",
-    "@types/chai-as-promised": "^7.1.2",
-    "@types/mocha": "^7.0.2",
-    "balena-config-karma": "^2.3.1",
-    "chai": "^4.2.0",
+    "@balena/lint": "^5.4.2",
+    "@types/chai": "^4.2.18",
+    "@types/chai-as-promised": "^7.1.4",
+    "@types/mocha": "^8.2.2",
+    "balena-config-karma": "^3.0.0",
+    "chai": "^4.3.4",
     "chai-as-promised": "^7.1.1",
-    "husky": "^4.2.5",
-    "karma": "^5.0.9",
-    "lint-staged": "^10.1.7",
-    "mocha": "^7.2.0",
+    "husky": "^6.0.0",
```

Comment:
Let's revert husky, since their new license is paid for non open-source projects.

Reply:
Oh I see that husky 6 is again free for open source :thinking: 
See: https://github.com/typicode/husky/releases/tag/v6.0.0

Let's confirm on FD.